src/vulkan/find_vulkan.cmake

```diff
@@ -19,17 +19,19 @@ set(Vulkan_FOUND FALSE)
 set(VULKAN_CTS_HEADER FALSE)
 set(VULKAN_LIB "")
 
-if (${AMBER_USE_LOCAL_VULKAN})
-  set(Vulkan_FOUND TRUE)
-  set(VulkanHeaders_INCLUDE_DIR
-    ${PROJECT_SOURCE_DIR}/third_party/vulkan-headers/include
-    CACHE PATH "vk headers dir" FORCE)
-  set(VulkanRegistry_DIR
-    ${PROJECT_SOURCE_DIR}/third_party/vulkan-headers/registry
-    CACHE PATH "vk_registry_dir" FORCE)
-  include_directories(BEFORE "${VulkanHeaders_INCLUDE_DIR}")
-  set(VULKAN_LIB vulkan)
-  message(STATUS "Amber: using local vulkan")
+if (NOT ${Vulkan_FOUND})
```

Comment:
It was a little mysterious to see this be conditioned this way.  But it follows the pattern in the rest of the file and is innocuous otherwise.  So ok with me.